src/options/options.html

```diff
@@ -1,70 +1,128 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+  <head>
+    <title>Orbit Extension Settings</title>
+    <link rel="stylesheet" href="./options.css" />
+    <script type="module" src="./options.js"></script>
+  </head>
+  <meta charset="utf-8" />
 
-<head>
-  <title>Orbit Extension Settings</title>
-  <link rel="stylesheet" href="./options.css" />
-  <script type="module" src="./options.js"></script>
-</head>
-
-<body>
-  <div class="flex-1 mx-auto w-full" style="width: 980px;">
-    <header class="flex items-center pb-3 pt-5 mt-2 mb-6">
-      <div class="mr-5">
-        <img alt="" src="../icons/orbit-logo-color-1x.png" srcset="
-              ../icons/orbit-logo-color-2x.png 2x,
-              ../icons/orbit-logo-color-3x.png 3x
-            " style="height: 30px; max-width: inherit;" />
-      </div>
-      <div class="mx-auto text-yellow-600"></div>
-      <div class="justify-end mx-2">
-        <a class="no-underline" href="https://app.orbit.love" target="_blank" rel="noopener">Open Orbit app</a>
-      </div>
-    </header>
-    <div class="pt-6 mx-auto w-full max-w-screen-sm">
-      <h1 class="mb-6">Orbit Extension Settings</h1>
-      <div x-data="orbit">
-        <button id="startOAuth" @click="startOAuthFlow" x-show="showLogin" class="mr-2">
-          Sign in with Orbit
-        </button>
-
-        <p x-show="!showLogin">
-          Successfully signed in! If you wish you sign out, visit your <a
-            href="https://app.orbit.love/user/settings/authorized_applications">User Settings on
-            Orbit</a> & revoke access for the "Orbit Browser Extension"
-          application.
-        </p>
+  <body>
+    <div class="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
+      <div class="mx-auto max-w-3xl sm:mt-10 lg:mt-20">
+        <div x-data="orbit">
+          <section>
```

Comment:
Here I’m using `<section>`s for the major “blocks” of the extension:

- The presentation text + login part
- Selecting a workspace